### PR TITLE
Update README.md

### LINKED ISSUE #504: docs: Update README for shouldi
https://github.com/intel/dffml/blob/master/examples/shouldi/README.md the docs need to be updated as `shouldi` has evolved since. There are many  new code analyzers wrapped by `shouldi ` which are missing in the README for example, npm_audit, go lang's linter etc. The TODO's mentioned are actually done .

### PR DESCRIPTION
**Pull request by TakshKamlesh**
FIxes #504 
Kindly inform what more could be updated / removed.

### CODE REVIEW

File: examples/shouldi/README.md [hunk 1]
```diff
@@ -29,9 +29,15 @@ Right now `shouldi` runs the following static analysis tools and complains if:
 
 - [safety](https://pyup.io/safety/)
   - Any issues are found
-- TODO: [bandit](https://pypi.org/project/bandit/)
+  
 - TODO: [pylint](https://pypi.org/project/pylint/)
```
Comment (pdxjohnny) [inline examples/shouldi/README.md:33]: Let's remove the bullet points about `pylint`

File: examples/shouldi/README.md [hunk 2]
```diff
@@ -29,9 +29,15 @@ Right now `shouldi` runs the following static analysis tools and complains if:
...
   - TDB (something about the number of errors)
+  
+Also , shouldi as been updated with following code wrappers:
```
Comment (pdxjohnny) [inline examples/shouldi/README.md:36]: - I don't think we need to say `Also, ...` here. We can just list all of them.
- Let's also remove the `Any issues are found` line below the `safety` bullet point

Comment (TakshKamlesh) [reply examples/shouldi/README.md:36]: OK I am working on it

Comment (TakshKamlesh) [reply examples/shouldi/README.md]: Done

**Comment by TakshKamlesh** [comment]
What's the issue with changelog file

**Pull request merged**